sims/vsim/Makefile

```diff
@@ -94,4 +94,4 @@ $(output_dir)/%.vpd: $(output_dir)/% $(sim_debug)
 #########################################################################################
 .PHONY: clean
 clean:
-	rm -rf $(build_dir) csrc $(sim_prefix)-* ucli.key vc_hdrs.h
+	rm -rf $(sim_dir)/generated-src/* csrc $(sim_prefix)-* ucli.key vc_hdrs.h
```

Comment:
I would define this in `variables.mk` so you can reuse it across the verilator and vcs sim directories. Which reminds me, can this be added in the verilator makefile also.